docs/topics/asset/asset-attributes-registry.md

```diff
@@ -0,0 +1,49 @@
+---
+description: AssetAttributesRegistry
+---
+
+# AssetAttributesRegistry
+
+## Introduction
+
```

Comment:
One sentence to introduce the concept of Asset and add a link to the Asset contract doc